csm_web/scheduler/views/section.py

```diff
@@ -488,3 +487,31 @@ def _student_add(self, request, section):
                 f"<Enrollment:Success> User {log_str(student.user)} enrolled in Section {log_str(section)}"
             )
             return Response({"id": student.id}, status=status.HTTP_201_CREATED)
+
+
+    @action(detail=True, methods=["put"])
+    def change_word_of_day(self, request, pk=None):
+        """
+        Request
+            data = {
+                'word_of_the_day': {new word of the day}
+            }
+        """
```

Comment:
I know this is very nitpicky, but change the docstring to the following to make it consistent with other functions:
"""
Updates the word of the day of a section occurrence.

Format of request.
            request.data is a dictionary with
                'word_of_the_day': new word of the day
"""